### PR TITLE
New package: ReorProject.Reor version 0.2.3

### LINKED ISSUE #151256: [Package Request]: Reor
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.reorproject.org/downloads
Publisher: [reorproject.org](https://reorproject.org/)
Package Name: Reor
Description: A self-organizing AI note-taking app that runs models locally.
Package Version: v0.2.3
Installer URL: https://github.com/reorproject/reor/releases/download/v0.2.3/Reor_0.2.3.exe

### PR DESCRIPTION
**Pull request by CoolPlayLin**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.2.1 :rocket:

- Resolves #151256

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/151258)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 877.)

File: manifests/r/ReorProject/Reor/0.2.3/ReorProject.Reor.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ReorProject.Reor
+PackageVersion: 0.2.3
+InstallerType: nullsoft
+UpgradeBehavior: install
+ReleaseDate: 2024-04-20
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/r/ReorProject/Reor/0.2.3/ReorProject.Reor.installer.yaml:10]:
```suggestion
- Architecture: x64
```

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

[This](https://www.npmjs.com/package/vectordb) appears to be the dependency. Unfortunately, not seeing it in our package manager yet:

```PowerShell
PS C:\ManVal> Find-WinGetPackage vectordb
PS C:\ManVal>
```

**Comment by CoolPlayLin** [comment]
Hi @samlhuillier 
Do you know where the issue is?

**Comment by CoolPlayLin** [comment]
It runs normally on my machine

**Comment by stephengillie** [comment]
> It runs normally on my machine

Is it because you already have `vectordb` installed? This seems to be the issue here - a missing dependency.

**Comment by CoolPlayLin** [comment]
CC @reorproject @chuyouchia @samlhuillier @rpdelaney

**Comment by samlhuillier** [comment]
Unfortunately this is an issue we've been having with Lancedb and their linking. The solution right now is to install [https://aka.ms/vs/17/release/vc_redist.x64.exe](https://github.com/microsoft/onnxruntime/issues/url) on the Windows machine (this installs the c++ runtime which can also be install by downloading and installing Visual Studio).

**Comment by rpdelaney** [comment]
@CoolPlayLin I have never seen this repo before. I assume you accidentally tagged me. Unsubscribing

**Comment by stephengillie** [comment]
Verify: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**